Google/resource-manager/project/locals.tf

```diff
@@ -5,13 +5,13 @@ locals {
   projects_components_specs  = lookup(local.projects_components, "specs", {})
   projects_components_common = lookup(local.projects_components, "common", {})
 
-  parent_folder = var.parent_folder == null ? lookup(local.projects_components_common, "folder_id", null) : var.parent_folder
-
   projects_specs = {
     for project, config in local.projects_components_specs :
         project => merge(local.projects_components_common, config)
   }
 
+  parent_folder = var.parent_folder == null ? lookup(local.projects_components_common, "folder_id", null) : var.parent_folder
```

Comment:
This could be the org, so shouldn't be called `_folder`

Reply:
Yes, we should be able to do that.

Reply:
Changed to have `parent_folder` and `parent_org` variables. Had noticed an issue that if someone were to set the `org_id` in `commons` but the `folder_id` in specs, terraform would try to assign both (which would fail). So updated the logic to ensure that doesn't happen. 
It was easier to get the logic working by having separate variables for both. Especially as the module should be taking just the id without the `folders/` or `organizations/` prefix.  
So the changes are:
* Created `local.projects_parents` with `parent_folder` and `parent_org` attributes (see [source](https://github.com/mesoform/Multi-Cloud-Platform-Foundations/blob/3b46586dca75c8d284ba771c04bb20e3ebedcdd4/Google/resource-manager/project/locals.tf#L15-L16))
  1. If the respective `var.parent_folder` or `var.parent_org` is set, the locals will take that value
  2. For `local.parent_folder` if `org_id` is set in the specs, it will be set to `null`, vice versa for `local.parent_org` 
  3. The `specs` block is checked for the relevant value, if that is null, the org is.
* Remove `"organizations/"` and `"folders/"` from IDs if present